tensorflow/compiler/xla/service/platform_util.h

```diff
@@ -61,11 +61,13 @@ class PlatformUtil {
   // Returns a vector of StreamExecutors for the given platform. The vector is
   // indexed by device ordinal (device numbering used by StreamExecutor). If an
   // element is nullptr, then the device is present by not supported by XLA.
+  // Optional parameter, allowed_devices controls which of the devices on the
+  // platform will have StreamExecutors constructed for. 
```

Comment:
No need to say it's an optional parameter, that is clear from the code.

>  allowed_devices controls which of the devices on the platform will have StreamExecutors constructed for. 

Reword sentence?

Reply:
Done.